ugandaemr-304.md

```diff
@@ -0,0 +1,58 @@
+## UgandaEMR 3.0.4 Release Notes
+
+Currently, work in progress
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Over Due for third DNS PCR
+    * Updated Criteria for Due and Over Due for Second DNA PCR
+ 2. HIV MER Indicator Reports 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnantal Register
+    
+    
+  
+### Fixes Made
+
+       1. Disabled the creation of Multiple encounters on the same visit date
```

Comment:
@solemabrothers This indentation is not necessary

Reply:
Done